app/main.py

```diff
@@ -1,6 +1,20 @@
+import os
+import requests
+from dotenv import load_dotenv
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    load_dotenv()
+    api_key = os.getenv("API_KEY")
+
+    URL = "http://api.weatherapi.com/v1/current.json"
+    PARAMS = {"key": api_key, "q": "Paris"}
+    response = requests.get(URL, params=PARAMS)
+    data = response.json()
```

Comment:
Consider adding error handling for the API request. You can check if the response status code indicates success and handle any exceptions that may occur during the request or JSON parsing.